R/reports.R

```diff
@@ -38,6 +53,17 @@ publish_rmd <- function(rmd_file, report_id=NULL, report_name=NULL,
                         provide_api_key=NULL, project_id=NULL, ...) {
```

Comment:
Not in this PR, but the docs for `project_id` says it's a bool, but I think it's a string

the "see also section" could use `\code{\link{publish_html}}` instead of `'publish_html'`